src/api/core/DatabaseServerlessHandler.ts

```diff
@@ -12,7 +12,7 @@ export abstract class DatabaseServerlessHandler<T> {
     protected abstract onHandleEvent(event?: T): Promise<ProxyResult>;
 
     public async execute(event: T): Promise<ProxyResult> {
-        this.initializeDatabase();
+        await this.initializeDatabase();
```

Comment:
> this function call is wrong, because the initializeDatabase() function is a async function.
@bigLucas 
In this case, just add an `await` it's enough?

Reply:
@Aristevao yes, it is!